client/components/telecom/telephony/abbreviatedNumbers/telecom-telephony-abbreviated-numbers.html

```diff
@@ -77,97 +77,37 @@
-                        <span class="word-break"
-                              data-ng-bind="abbreviated.surname"></span>
-                    </td>
-                    <td>
-                        <span class="word-break"
-                              data-ng-bind="abbreviated.name"></span>
-                    </td>
-                    <td>
-                        <oui-spinner data-ng-if="abbreviated.removing"
-                                     data-size="s">
-                        </oui-spinner>
-                        <div class="btn-group float-right"
-                             data-uib-dropdown
-                             data-dropdown-append-to-body>
-                            <button id="btn-sms-abbreviated-numbers-action-{{ index }}"
-                                    type="button"
-                                    class="btn btn-link"
-                                    data-uib-dropdown-toggle>
-                                <span class="ovh-font ovh-font-dots mr-2" aria-hidden="true"></span>
-                                <span class="sr-only"
-                                      data-translate="common_actions">
-                                </span>
-                            </button>
-                            <ul class="dropdown-menu dropdown-menu-right"
-                                role="menu"
-                                aria-labelledby="btn-sms-abbreviated-numbers-action-{{ index }}"
-                                data-uib-dropdown-menu>
-                                <li role="menuitem">
-                                    <i class="ovh-font ovh-font-editer mr-2" aria-hidden="true"></i>
-                                    <button type="button"
-                                            class="btn btn-link"
-                                            data-ng-disabled="abbreviated.updating"
-                                            data-ng-click="$ctrl.update(abbreviated)"
-                                            data-translate="edit">
-                                    </button>
-                                </li>
-                                <li role="menuitem">
-                                    <i class="ovh-font ovh-font-bin mr-2" aria-hidden="true"></i>
-                                    <button type="button"
-                                            class="btn btn-link"
-                                            data-ng-disabled="abbreviated.removing"
-                                            data-ng-click="$ctrl.remove(abbreviated)"
-                                            data-translate="delete">
-                                    </button>
-                                </li>
-                            </ul>
-                        </div>
-                    </td>
-                </tr>
-            </tbody>
-        </table>
-    </div>
+    <oui-datagrid data-rows="$ctrl.paginatedAbbreviatedNumbers">
```

Comment:
Hello, 
In order to avoid confusion, it could be good to call your paginatedAbbreviatedNumbers variable : abbreviatedNumbers

Reply:
Indeed, rows does not take paginated data.